tests/fixture-ts/app/initializers/test-init.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type Application from '@ember/application';
+
+export default {
+  name: 'test-init',
+  initialize(application: Application) {
+    (application as any).__test_init = 'coming from the initializer';
```

Comment:
What about the use of 'unknown' instead of any? Fewer eslint rules also.

> (application as unknown as { __test_init?: string }).__test_init = 'coming from the initializer';

Reply:
Unknown can't have properties set on it.

But also, the lints are for users, and much less so us making sure we wired the tools up correct

Reply:
Thanks for the clarification!